scalafix-tests/output/src/main/scala-3/test/removeUnused/RemoveUnusedParams.scala

```diff
@@ -0,0 +1,9 @@
+package test.removeUnused
+
+object UnusedParams {
+  val f: String => Unit = _ => println("f")
+  val ff = (_: String) => println("f")
+  val fs = (used: String, _: Long) => println(used)
+  def g(x: String => Unit): Unit = ???
+  g{implicit string => println("g")}
```

Comment:
there is no diagnostic for this on the semanticdb emitted by scala 3.7.0

Reply:
Looks like a regression from the huge PR about unused settings. I think this might not only be semanticdb.